test/runtests.jl

```diff
@@ -46,20 +47,96 @@ end
 #### scalar transformations correctness checks
 ####
 
+@testset "scalar show" begin
+    @test string(asℝ) == "asℝ"
+    @test string(asℝ₊) == "asℝ₊"
+    @test string(asℝ₋) == "asℝ₋"
+    @test string(as𝕀) == "as𝕀"
+    @test string(TVShift(0) ∘ TVNeg() ∘ TVExp()) == "asℝ₋" 
+    @test string(TVShift(0) ∘ TVExp()) == "asℝ₊" 
+    @test string(TVShift(0) ∘ TVScale(1) ∘ TVLogistic()) == "as𝕀" 
+    @test string(TVNeg() ∘ TVExp()) == "asℝ₋" 
+    @test string(∘(TVExp())) == "asℝ₊" 
+    @test string(∘(TVLogistic())) == "as𝕀" 
+    @test string(as(Real, 0.0, 2.0)) == "as(Real, 0.0, 2.0)"
+    @test string(as(Real, 1.0, ∞)) == "as(Real, 1.0, ∞)"
+    @test string(as(Real, -∞, 1.0)) == "as(Real, -∞, 1.0)"
+
+    @test string(TVShift(4.0)) == "TVShift(4.0)"
+    @test string(TVScale(4.0)) == "TVScale(4.0)"
+    @test string(TVExp()) == "TVExp()"
+    @test string(TVLogistic()) == "TVLogistic()"
+    @test string(TVNeg()) == "TVNeg()"
+
+    @test string(TVScale(2.0) ∘ TVNeg() ∘ TVExp()) == "TVScale(2.0) ∘ TVNeg() ∘ TVExp()" 
+    @test string(TVScale(5.0u"m") ∘ TVExp()) == "TVScale(5.0 m) ∘ TVExp()" 
+end
+
 @testset "scalar transformations consistency" begin
     for _ in 1:100
         a = randn() * 100
         test_transformation(as(Real, -∞, a), y -> y < a)
         test_transformation(as(Real, a, ∞), y -> y > a)
         b = a + 0.5 + rand(Float64) + exp(randn() * 10)
         test_transformation(as(Real, a, b), y -> a < y < b)
+
+        posexp = TVShift(a) ∘ TVExp()
+        negexp = TVShift(a) ∘ TVNeg() ∘ TVExp()
+        finite = TVShift(a) ∘ TVScale(b-a) ∘ TVLogistic()
+        test_transformation(posexp, y -> y > a)
+        test_transformation(negexp, y -> y < a)
+        test_transformation(finite, y -> a < y < b)
     end
     test_transformation(as(Real, -∞, ∞), _ -> true)
 end
 
+@testset "scalar non-Real (Unitful) consistency" begin
+    for _ in 1:10    
+        a = randn() * 100
+        b = a + 0.5 + rand(Float64) + exp(randn() * 10)
+        t1 = TVScale(2u"m") ∘ TVShift(a) ∘ TVExp() 
+        test_transformation(t1, y -> y > a*2u"m", jac=false)
+        t2 = TVScale(1u"s") ∘ TVShift(a) ∘ TVScale(b-a) ∘ TVLogistic()
+        test_transformation(t2, y -> (a*u"s" < y < b*u"s"), jac=false)
```

Comment:
maybe a test that it errors for `transform_and_logjac`

Reply:
I did have this in another test set, but if you naturally expected that here, this is probably the more logical place. Moved to here